File: Conceptos Basicos/ejercicio02.java
//2. Dado un arreglo de números , calcula la suma total.
import java.util.Scanner;

public class ejercicio02 {
    public static void main (String[] arg){
        int array[] = new int [5];
        Scanner scan = new Scanner(System.in);
        for (int i =0;i<array.length ; i++ ){
            System.out.print("Ingresa el digito ["+i+"]:");
            array [i]= scan.nextInt();
        }

        int total = 0;
        for (int j =0;j<array.length ; j++ ){
            total= total+ array[j];
        }

        System.out.println("El total es "+total);

    }
}

File: Conceptos Basicos/ejercicio02.py
#2. Dado un arreglo de números , calcula la suma total.
array = []
for i in range (0,5): 
    array.append(int(input(f"Digita un numero en la posicion [{i}]: \n")))

total = 0
for i in range(len(array)):
    total= total + array[i]

print("El total del arreglo fue:",total)